librad/src/meta/project.rs

```diff
@@ -38,8 +38,11 @@ pub fn default_branch() -> String {
     DEFAULT_BRANCH.into()
 }
 
-#[derive(Clone, Default, Deserialize, Serialize, Debug, PartialEq)]
+#[derive(Clone, Deserialize, Serialize, Debug, PartialEq)]
 pub struct ProjectInfo {
+    // Marker so `EntityInfo` can deserialize correctly
```

Comment:
Can you explain this trickery?

Reply:
See above...

Reply:
The above being: https://github.com/radicle-dev/radicle-link/pull/170#discussion_r438919403